### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Aneesh02**
@KShivendu Please look at the ReadMe and suggest changes if any. Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
 # Photo-Sharing-App
+
+## About the Project
+Photo Sharing App is a web app that allows a user to get group photos that include them without having to keep or sort through all of the photos, saving them time and
+storage space. When a photo is uploaded, the app identifies the faces in it and arranges them according to the users who have been identified. Each identified user gets
+a separate album with all their photos allowing them to get the photos they need seamlessly.
 Upload group photos, arrange them by faces using ML and share them with ease. 🤳🤖
```
Comment (KShivendu) [inline README.md:7]: You may add some implementation details as well (briefly)

Comment (KShivendu) [reply README.md:7]: Something like a "How does it work?" section.

Comment (Aneesh02) [reply README.md:7]: Added

File: README.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
...
+
+![5](https://user-images.githubusercontent.com/72318258/150670162-4a8ae9d4-34a3-4c25-8e64-8442c0cd36ba.png)
+This is Ron Weasley a.k.a Rupert Grint's album. Out of the original 6 images he was present in just 4 and now we can download these pics indiviually or as a zip.
+
+
+## Frequently Asked Questions
+
+**Q: I got Error: That port is already in use.**
+
+A: It occurs because some process has occupied that port. You may kill the process using ``sudo fuser -k 8000/tcp`` or use a different port.
```
[the remark this answers is not in the document]
Comment (Aneesh02) [reply README.md]: Done

**Comment by Aneesh02** [comment]
@KShivendu Re-review it. Thanks

**Comment by KShivendu** [comment]
@Aneesh02 I believe these changes have been already covered in the other PR. If so, please close this one.

**Pull request closed without merging**